src/server.ts

```diff
@@ -241,23 +244,24 @@ export class InversifyExpressServer {
     }
 
     private handlerFactory(
-        controllerName: any,
+        controllerName: string,
         key: string,
         parameterMetadata: Array<interfaces.ParameterMetadata>,
     ): express.RequestHandler {
         return async (req: express.Request, res: express.Response, next: express.NextFunction) => {
             try {
                 const args = this.extractParameters(req, res, next, parameterMetadata);
-
                 const httpContext = this._getHttpContext(req);
                 httpContext.container.bind<interfaces.HttpContext>(TYPE.HttpContext)
                 .toConstantValue(httpContext);
-
                 // invoke controller's action
+
+                // TODO: Cast this some how ??
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
                 const value = await httpContext.container.getNamed<any>(
                     TYPE.Controller,
                     controllerName,
-                )[key](...args);
+                )?.[key](...args);
```

Comment:
Regarding this `any`, just add some typings. I'm adding a code sample, I do not exactly know which are the right types here, but my approach is the following one:

```ts
type ControllerHandler = (...params: Array<unknown>) => unknown;
type Controller = Record<string, ControllerHandler>;

const service: Controller = httpContext.container.getNamed<Controller>(
    TYPE.Controller,
    controllerName,
);

const controllerHandler: ControllerHandler = service[key] as ControllerHandler;

const value: unknown = await controllerHandler(...args);
```

keep in mind `ControllerHandler` and `Controller` may not be the right types